scripts/create_source_tarball.sh

```diff
@@ -25,9 +25,9 @@ REPO_ROOT="$(dirname "$0")"/..
     git checkout-index -a --prefix="$SOLDIR"
     # Store the commit hash
     echo "$commithash" > "$SOLDIR/commit_hash.txt"
-    if [ -e prerelease.txt ] && [ ! -s prerelease.txt ]
+    if [ ! -e cmake/prerelease.lock ]
     then
-        cp prerelease.txt "$SOLDIR/"
+        cp cmake/prerelease.lock "$SOLDIR/"
```

Comment:
This file will now be stored in the repository so the `git checkout-index` command should already copy it.

Reply:
But anyway, we no longer need this script. You can replace most of its content with the `git archive` command.

Reply:
Should we just delete the file?

Reply:
I think it would be fine to delete it completely. I don't think we were using it for anything other that release checklist (@nishant-sachdeva please do a search for `create_source_tarball.sh` to make sure) and I doubt people are building their own source packages.

Reply:
I've run the command
`git grep "create_source_tarball.sh" ` on the git repo and I haven't found any references to this file.

So, deleting it looks good to me